### DISCUSSION TITLE
In the cluster mode,How is Kafka Fetcher configured？

### DISCUSSION THREAD

**Discussion by cuixianyu-github** [apache/skywalking#10638] [Q&A]
https://skywalking.apache.org/docs/main/next/en/setup/backend/kafka-fetcher/
What is the configuration of consumePartitions parameters

**Comment by wu-sheng** [comment]
If you feel hard to understand this, read some Kafka documents. There aren't many new things from SkyWalking's perspective.

**Comment by cuixianyu-github** [comment]
Thank you very much for your reply. Actually, what I want to ask is how the oap cluster is configured？1 or 2
1、---------------
kafka-fetcher:
  selector: ${SW_KAFKA_FETCHER:default}
  default:
    bootstrapServers: ${SW_KAFKA_FETCHER_SERVERS:localhost:9092}
    namespace: ${SW_NAMESPACE:""}
    partitions: ${SW_KAFKA_FETCHER_PARTITIONS:3}
    replicationFactor: ${SW_KAFKA_FETCHER_PARTITIONS_FACTOR:2}
    isSharding: ${SW_KAFKA_FETCHER_IS_SHARDING:false}
    consumePartitions: ${SW_KAFKA_FETCHER_CONSUME_PARTITIONS:""}
    consumers: ${SW_KAFKA_FETCHER_CONSUMERS:1}
   2、 -------------------------
    kafka-fetcher:
  selector: ${SW_KAFKA_FETCHER:default}
  default:
    bootstrapServers: ${SW_KAFKA_FETCHER_SERVERS:localhost:9092}
    namespace: ${SW_NAMESPACE:""}
    partitions: ${SW_KAFKA_FETCHER_PARTITIONS:3}
    replicationFactor: ${SW_KAFKA_FETCHER_PARTITIONS_FACTOR:2}
    isSharding: ${SW_KAFKA_FETCHER_IS_SHARDING:true}
    consumePartitions: ${SW_KAFKA_FETCHER_CONSUME_PARTITIONS:1,3,5}
    consumers: ${SW_KAFKA_FETCHER_CONSUMERS:1}
    kafkaConsumerConfig:
      enable.auto.commit: true
